official/docs/java/endShipper/CreateEndShipper.java

```diff
@@ -0,0 +1,25 @@
+import com.easypost.EasyPost;
+import java.util.HashMap;
+
+public class CreateEndShipper {
+    public static void main(String[] args) {
+        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
+
+        Map<String, Object> endShipperData = new HashMap<String, Object>();
```

Comment:
```suggestion
        HashMap<String, Object> endShipperData = new HashMap<String, Object>();
```

This will need to be HashMap now